BankApplication/src/com/luxoft/bankapp/bank_application/TestSerialization.java

```diff
@@ -0,0 +1,30 @@
+package com.luxoft.bankapp.bank_application;
+
+import com.luxoft.bankapp.model.Account;
+import com.luxoft.bankapp.model.CheckingAccount;
+import com.luxoft.bankapp.model.Client;
+import com.luxoft.bankapp.model.Gender;
+import com.luxoft.bankapp.model.SavingAccount;
+import com.luxoft.bankapp.service.BankFeedService;
+import com.luxoft.bankapp.service.BankFeedServiceImpl;
+import com.luxoft.bankapp.service.BankServiceImpl;
+
+public class TestSerialization {
+
+	public static void main(String[] args) {
```

Comment:
Why in a world test has a main method? If you are testing use unit test framework. This will allow to run tests automatically later and not just during engineering time.